docs/src/SemCor.md

```diff
@@ -15,70 +15,65 @@ In the demo below is is shown how one might index the corpus to get a collection
 ```julia
 
```

Comment:
good catch, this became outdated, with the InternedStrings changes

Reply:
I feel now, that I should have commented that fact while updating. :sweat_smile: 
Its my bad.